<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info_dialog</class>
 <widget class="QDialog" name="info_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>info_dialog</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>141</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_progress">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>419</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Continuar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>419</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Continuar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
